components/tray/index.css

```diff
@@ -10,70 +10,96 @@ OF ANY KIND, either express or implied. See the License for the specific languag
   overflow: auto;
   outline: none;
-  border-radius: var(--spectrum-tray-full-width-border-radius)
-    var(--spectrum-tray-full-width-border-radius)
-    var(--spectrum-tray-border-radius) var(--spectrum-tray-border-radius);
-  padding: var(--spectrum-tray-padding-y) var(--spectrum-tray-padding-x);
   /* Start offset by the animation distance */
   transform: translateY(100%);
   /* Exit animations */
-  transition: opacity var(--spectrum-dialog-confirm-exit-animation-duration)
+  transition: opacity var(--mod-tray-exit-animation-duration, var(--spectrum-tray-exit-animation-delay))
       cubic-bezier(0.5, 0, 1, 1)
-      var(--spectrum-dialog-confirm-exit-animation-delay),
+      var(--mod-tray-exit-animation-delay, var(--spectrum-tray-exit-animation-delay)),
     visibility 0ms linear
       calc(
-        var(--spectrum-dialog-confirm-exit-animation-delay) +
-          var(--spectrum-dialog-confirm-exit-animation-duration)
+        var(--mod-tray-exit-animation-delay, var(--spectrum-tray-exit-animation-delay)) +
+          var(--mod-tray-exit-animation-duration, var(--spectrum-tray-exit-animation-duration))
       ),
-    transform var(--spectrum-dialog-confirm-exit-animation-duration)
+    transform var(--mod-tray-exit-animation-duration, var(--spectrum-tray-exit-animation-duration))
       cubic-bezier(0.5, 0, 1, 1)
-      var(--spectrum-dialog-confirm-exit-animation-delay);
+      var(--mod-tray-exit-animation-delay, var(--spectrum-tray-exit-animation-duration));
+
+  background-color: var(--highcontrast-tray-background-color, var(--mod-tray-background-color, var(--spectrum-tray-background-color)));
 
   &.is-open {
     @inherit: %spectrum-overlay--open;
     /* Entry animations */
     transition: transform
-        var(--spectrum-dialog-confirm-entry-animation-duration)
+        var(--mod-tray-entry-animation-duration, var(--spectrum-tray-entry-animation-duration))
         cubic-bezier(0, 0, 0.4, 1)
-        var(--spectrum-dialog-confirm-entry-animation-delay),
-      opacity var(--spectrum-dialog-confirm-entry-animation-duration)
+        var(--mod-tray-entry-animation-delay, var(--spectrum-tray-entry-animation-delay)),
+      opacity var(--spectrum-tray-entry-animation-duration, var(--mod-tray-entry-animation-duration))
         cubic-bezier(0, 0, 0.4, 1)
-        var(--spectrum-dialog-confirm-entry-animation-delay);
+        var(--mod-tray-entry-animation-delay, var(--spectrum-tray-entry-animation-delay));
     transform: translateY(0);
   }
 }
 
-/* Should match --spectrum-tray-max-width above */
-@media (max-width: 375px) {
-  .spectrum-Tray {
-    border-radius: var(--spectrum-tray-border-radius);
-  }
+.spectrum-Tray-overlay {
+  inset-inline-start: 0;
+  /* Fixed to the bottom of the viewport */
+  position: fixed;
+  bottom: 0;
+
+  display: flex;
+  justify-content: center;
+  inline-size: 100%;
+
+  --spectrum-tray-overlay-background-color: var(--spectrum-overlay-color);
```

Comment:
Can you move this custom property definition from here to being defined under `.spectrum-Tray`? For a consumer to customize this in SWC, it needs to be defined at the root.

Reply:
This won't work if I move it to .spectrum-Tray since it isn't a modifier class and needs to be in container that wraps .spectrum-Tray. Is there another way to make this work? I'm not familiar with how SWC consumes components.

Reply:
I removed overlay styles as they will likely be included with Modal.